packs/data/the-enmity-cycle-bestiary.db/the-enmity-cycle.json

```diff
@@ -56,7 +56,7 @@
             },
             "stealth": {
                 "details": "<p>(expert) or <em>@UUID[Compendium.pf2e.spells-srd.Detect Magic]{Detect Magic}</em></p>",
-                "value": 6
+                "value": 16
```

Comment:
Doesn't the system store this -10?

Reply:
As a complex hazard, it rolls for initiative, so it needs to be +16